implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -140,10 +140,14 @@ protected void publish() {
             return;
         }
 
-        for (List<Meter> batch : MeterPartition.partition(this, config.batchSize())) {
+        // Stackdriver's API limits us to less than 200 events per call
+        int limit = 199;
+        for (List<Meter> batch : MeterPartition.partition(this, Math.min(config.batchSize(), limit))) {
```

Comment:
We don't need this partitioning anymore since the subsequent grouping is already splitting the `TimeSeries`.

Reply:
Good catch, thanks, fixed in c99df29b4a30cfb318195241eef0f6e72d01e751.

Reply:
I mean I don't think we need to use MeterPartition at all with Stackdriver anymore, since we are batching the `TimeSeries` before sending. I can fix this while merging, though.